src/components/LayoutTemplates/UNSTABLE__DetailsPage1/index.js

```diff
@@ -0,0 +1,20 @@
+/**
+ * @file Details page template 1 entry point.
+ * @copyright IBM Security 2020
+ */
+
+import UNSTABLE__DetailsPage1 from './UNSTABLE__DetailsPage1';
```

Comment:
For consistency, would you mind following the same import syntax used in `LayoutTemplates/index` ?

Reply:
So speaking of this, was thinking it would be better to start doing `export { default as ComponentName } from 'path/to/component'`. Would clear syntax errors, follow Carbon's format, and also address this comment: https://github.com/carbon-design-system/ibm-security/pull/315#discussion_r367624887